### PR TITLE
PR Create component for icons #119 - Feature/35743

### PR DESCRIPTION
**Pull request by anqaka**
#119 
https://dev.snowdog.pro/issues/35743

### CODE REVIEW

File: components/01-globals/icon/_icon.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+$icon-width       : 48px !default;
+$icon-padding     : 12px !default;
```
Comment (Januszpl) [inline components/01-globals/icon/_icon.scss]: Shouldn't we use $icon__width and $icon__padding? (as I see this approach in other files) (as class names are not 1:1 to variables names) (check _badge.scss or _header.scss)

File: components/01-globals/icon/_icon.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+$icon__svg-width  : 24px !default;
+$icon__svg-height : 24px !default;
+$icon__svg-padding: 2px !default;
+
+.icon {
+    padding: $icon-padding;
+    display: flex;
+    flex-flow: row nowrap;
```
Comment (Januszpl) [inline components/01-globals/icon/_icon.scss]: why we use here flex?

Comment (anqaka) [reply components/01-globals/icon/_icon.scss]: to inline the `svg` insine `i` vertically. I can also use `display: inline-block` and `vertical-align: middle`. But @Igloczek said that we shouldn't use it. I'm not sure which one is better...

File: components/01-globals/icon/_icon.scss (hunk 1 above)
Comment (anqaka) [reply components/01-globals/icon/_icon.scss]: You're right, thnx, fixed

File: components/01-globals/icon/_icon.scss (hunk 2 above)
Comment (anqaka) [reply components/01-globals/icon/_icon.scss]: after our conversation I see that `display: block` is enough. Thanks, corrected :)

**Pull request merged**